scripts/update-tracing-runtime-lock.sh

```diff
@@ -13,7 +13,4 @@ echo
 cargo fetch
 echo 
 
-echo "Removing shared directory in tracing/${VERSION}"
-rm -rf shared
-
-cd ../..
\ No newline at end of file
+cd ../..
```

Comment:
Not sure why this was done but it made sense that it was removed, else next time you would commit the duplicated shared repo that was copied to `cargo fetch`

Reply:
the shared library is linked temporarily but then removed, and it causes the check on the source code to fail